scripts/email_registration_contributors.py

```diff
@@ -0,0 +1,64 @@
+"""Script for sending OSF emails to all users who contribute to registrations."""
+
+import datetime
+import logging
+
+from modularodm import Q
+
+from website import models
+from website.app import init_app
+from website.mails import Mail, send_mail
+
+from scripts import utils as script_utils
+
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+MESSAGE_NAME = 'retraction_and_embargo_addition'
+MAILER = Mail(
+    'email_contributors_of_registrations',
+    'Important Update to Registrations on OSF'
+)
+
+
+def send_retraction_and_embargo_addition_message(contrib, label, mail, dry_run=True):
+    if label in contrib.security_messages:
+        return
+    if not dry_run:
+        send_mail(contrib.username, mail, user=contrib)
+        contrib.security_messages[MESSAGE_NAME] = datetime.datetime.utcnow()
+        contrib.save()
+
+
+def get_registration_contributors():
+    """Returns set of users that contribute to registrations."""
+    registrations = models.Node.find(Q('is_registration', 'eq', True))
+    contributors = []
+
+    def has_received_message(contrib):
+        query = (Q('_id', 'eq', contrib._id) & Q('security_messages.{0}'.format(MESSAGE_NAME), 'exists', False))
+        return False if len(models.User.find(query)) > 0 else True
```

Comment:
it's actually the compliment of that -- I think `return models.User.find(query).count() == 0` is more concise yet still clear. `return not bool(models.User.find(query).count())` just looked weird